tests/jerry/regression-test-issue-736.js

```diff
@@ -31,7 +31,7 @@ try {
 }
 
 try {
-  eval("var x; x instanceof assert;");
+  eval("var x = {}; x instanceof assert;");
```

Comment:
This looks like an unrelated change (bug fix of a test). Shouldn't this be a separate commit? If you discovered the issue after adding the new functionality in this patch then it can be part of this commit but should be mentioned in the commit message.